libsawtooth/Cargo.toml

```diff
@@ -75,7 +75,7 @@ pending-batch-queue = ["validator-internals"]
 receipt-store = ["stores"]
 redis-store = ["redis", "stores"]
 stores = []
-transaction-receipt-store = []
+transaction-receipt-store = ["lmdb-zero"]
```

Comment:
It might be better to have an additional feature `"transaction-receipt-store-lmdb"`

Reply:
added a `transaction-receipt-store-lmdb` feature